src/jdk.charsets/share/classes/sun/nio/cs/ext/ISO2022_CN.java

```diff
@@ -86,17 +85,13 @@ public boolean canEncode() {
         private boolean shiftOut;
         private byte currentSODesig;
 
-        private static final Charset gb2312 = new EUC_CN();
-        private static final Charset cns = new EUC_TW();
-        private final DoubleByte.Decoder gb2312Decoder;
-        private final EUC_TW.Decoder cnsDecoder;
+        private static final DoubleByte.Decoder gb2312Decoder =
```

Comment:
Since this is a shared constant, I'd use all caps field name.

Reply:
Will fix.